src/commands/deploy/title-escrow.ts

```diff
@@ -0,0 +1,71 @@
+import { Argv } from "yargs";
```

Comment:
I handled the case where people dont add 0x, maybe u want to do it as well (I dont mind and I didnt do it everywhere to)

https://github.com/Open-Attestation/open-attestation-cli/blob/master/src/commands/document-store/issue.ts#L51

Reply:
I don't think that should be done for eth addresses since all of them starts with 0x. That said, it might be valuable to add in a validator for addresses

Reply:
added validator for address